.coveragerc

```diff
@@ -0,0 +1,39 @@
+# .coveragerc to control coverage.py
+[run]
+branch = True
+omit = 
+    #omit anything in .local directory anywhere
+    */.local/*
+    #omit everything in /usr
+    */usr/*
+    #omit everthing in /Users   
+    */Users/*
+
+source =
+    .
+
+[report]
+omit =
+    */Users/mbarakmbigo/virtualenvs/*
```

Comment:
This is too user specific. This will only run on your system. you might want to use *virtualenvs/* instead